code/game/objects/items/devices/camera_bug.dm

```diff
@@ -25,6 +25,7 @@
 	integrated_console = new(src)
 	integrated_console.parent = src
 	integrated_console.network = list("SS13", "camera_bug[UID()]")
+	GLOB.restricted_camera_networks += "camera_bug[UID()]"
```

Comment:
I don't think this actually prevents the AI from seeing them, it just prevents them from jumping to it from what I can tell.